vendor/k8s.io/system-validators/validators/types_unix.go

```diff
@@ -48,11 +49,14 @@ var DefaultSysSpec = SysSpec{
 			{Name: "PROC_FS"},
 			{Name: "NETFILTER_XT_TARGET_REDIRECT", Aliases: []string{"IP_NF_TARGET_REDIRECT"}},
 			{Name: "NETFILTER_XT_MATCH_COMMENT"},
+			{Name: "FAIR_GROUP_SCHED"},
```

Comment:
WRT the new required fields:
https://github.com/kubernetes/kubeadm/issues/2335

@neolit123 

> both these PRs introduce new required configuration options (FAIR_GROUP_SCHED and CGROUP_PIDS), which can trip existing setups:
kubernetes/system-validators#19
kubernetes/system-validators#20
how common are these options and do you have any concerns about that?
i'm less concerned about CGROUP_PIDS, for some reason.

@odinuge 

> It has never been supported (or been working) running with a configuration that fails with the new validation, so I think it should be good to go. The only places they might be disabled, from my experience, are raspberry-pi like devices used for prototyping.

Reply:
can you clarify what the implications of "both these PRs introduce new required configuration options" are? does this mean an existing configuration will no longer work?

Reply:
@odinuge confirmed that kernels without these options (now required by the library) never worked with the kubelet.